jujugui/static/gui/src/app/components/mid-point/_mid-point.scss

```diff
@@ -0,0 +1,44 @@
+.mid-point {
+    color: $cool-grey;
+    padding: 30px 20px;
+
+    &__title {
+        margin-bottom: 20px;
+        font-size: 22px;
+    }
+
+    &__charm-list {
+        display: flex;
+        // The list items have margin right and we don't want a double gap on
+        // the right.
+        margin: 0 -20px 0 0;
```

Comment:
Oh, I tried with last-child and for some reason it wasn't working and I assumed it was a flex issue, but I just tried again and now it works!